packages/expo-ui/ios/Gauge/Gauge.swift

```diff
@@ -0,0 +1,53 @@
+// Copyright 2025-present 650 Industries. All rights reserved.
+
+import SwiftUI
+import ExpoModulesCore
+
+struct GaugeView: ExpoSwiftUI.View {
+  @EnvironmentObject var props: GaugeProps
+  @EnvironmentObject var shadowNodeProxy: ExpoSwiftUI.ShadowNodeProxy
+
+  var body: some View {
+#if !os(tvOS)
+    if #available(iOS 16.0, *) {
+      let range = (props.min?.value ?? 0.0)...(props.max?.value ?? 1.0)
+      ExpoSwiftUI.AutoSizingStack(shadowNodeProxy: shadowNodeProxy, axis: .both) {
+        Gauge(value: props.current.value, in: range) {
+          if let label = props.label {
+            if let color = props.labelColor {
+              Text(label).foregroundColor(color)
+            } else {
+              Text(label)
+            }
```

Comment:
Not sure if it will work (i haven't used the .let view builder yet) but can you try:
```suggestion
            Text(label)
              .let(color) { $0.foregroundColor($1) } 
```

Reply:
It worked! Thanks